addon/components/sortable-group.js

```diff
@@ -18,29 +19,35 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get() { return a(); }
```

Comment:
Ahh ok. I didn't know that

The deprecation was caused by this line:

`y: computed(function(_, value) {`

And since I was updating that one I figured may as wel update all of them.. Wasn't aware it was the argument length that caused the deprecation..

Good to know.. That will change the way I do some of my computed properties in the future.